docs/admin_design.md

```diff
@@ -0,0 +1,17 @@
+# Category Design for Admin-Web
+This document describes how to achieve the admin-web [requirement](./admin-requirement.md).
+
+## 1. Functions
+For admin-web, category service provide following functions
++ create a new category
++ delete category
++ update attribute of category
++ get individual category 
++ get all categories in order
+
+## 2. Workflow
+
+### 2.1. Creating a new category
+![create a new category](./resources/create_a_new_category.png)
```

Comment:
Use word to describe your work flow first, other people can not modify your image